chromium_src/chrome/common/url_constants.cc

```diff
@@ -202,6 +202,9 @@ const char kSyncGoogleDashboardURL[] =
 const char kSyncLearnMoreURL[] =
     "https://support.brave.com/";
 
+const char kSigninInterceptManagedDisclaimerLearnMoreURL[] =
+    "https://support.google.com/chrome/a/?p=profile_separation";
```

Comment:
why do we need this

Reply:
Usually we replace with support.brave.com, we should probably do the same here

Reply:
Changed in 006ca57436

Reply:
yeah sorry, i was asking why it links to google.